umh.docs.umh.app/content/en/docs/features/postgresql_and_grafana_alerts.md

```diff
@@ -0,0 +1,245 @@
++++
+title = "Alerts in Grafana using PostgreSQL"
+menuTitle = "Alerts in Grafana using PostgreSQL"
+description = ""
+weight = 50
+draft = false
++++
+
+The United Manufacturing Hub utilizes a TimeScaleDB database which is based on
+PostgreSQL. Therefore, you can use the PostgreSQL plugin in Grafana to implement 
+alerts. This step-by-step tutorial will guide you through the implementation of 
+the PostgreSQL plugin, the creation of an alert rule, and its connection to the
+desired contacts and services.
+
+
+## Why should I use alerts?
+
+  Alerts in Grafana offer real-time monitoring and proactive problem detection, 
+  enabling users to swiftly identify and address performance issues or system 
+  anomalies. By leveraging custom threshold-based notifications, teams can
+  enhance operational efficiency, reduce downtime, and maintain optimal system
+  health.
+
+## How can I implement alerts in Grafana?
+
+  As an example, this tutorial will use the simulated OPC-UA data from this
+  [tutorial](https://learn.umh.app/course/creating-a-node-red-flow-with-simulated-opc-ua-data/).
+  You need a running instance of the United Manufacturing Hub Stack with data
+  (like in the linked tutorial) to create an alert.
+
+### Important Expressions
+
+- **Alert rule:** Define the event and its duration that should trigger the 
+    alert.
+- **Contact point:** Define all the contacts and the used services (for this 
+    example, Discord).
+- **Notification policy:** Link the contact points to the alert rules.
+- **Label:** Labels are tags for the alerts. For example different teams, 
+  factories, production lines or priorities.
+- **Mute time and Silence:** You can mute an alert on a regular basis
+  or a singular, planned time frame (for example maintenance).
+
+### Installation and SetUp
```

Comment:
maybe its better to put the exact tutorial on learn.umh.app incl. images. here you can then only reference the overall approach "Setup PostgreSQL data source, go to Grafana alerting, create alert, add alert notifier" and then reference the detailed tutorial

Reply:
OK, makes sense. I will have a look tomorrow, how it can be split up in a usefull way.